corehq/apps/app_manager/static/app_manager/js/add_ons.js

```diff
@@ -1,8 +1,17 @@
 "use strict";
-hqDefine("app_manager/js/add_ons", function () {
-    var sectionChanger = hqImport("app_manager/js/section_changer"),
-        initialPageData = hqImport("hqwebapp/js/initial_page_data");
-
+hqDefine("app_manager/js/add_ons", [
+    'jquery',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/bootstrap3/main',
+    'app_manager/js/section_changer',
```

Comment:
This script (add_ons) is included here: https://github.com/dimagi/commcare-hq/blob/5fe14801267a27d02a155896edf9171583f0cba9/corehq/apps/app_manager/templates/app_manager/app_view.html#L28

The section_changer has already been loaded in the parent template, here: https://github.com/dimagi/commcare-hq/blob/5fe14801267a27d02a155896edf9171583f0cba9/corehq/apps/app_manager/templates/app_manager/apps_base.html#L73